extensions/blocks/simple-payments/featured-media.js

```diff
@@ -23,7 +24,7 @@ export default ( { featuredMediaId, featuredMediaUrl, featuredMediaTitle, setAtt
 	if ( ! featuredMediaId ) {
 		return (
 			<MediaPlaceholder
-				icon="format-image"
+				icon={ <BlockIcon icon={ icon } /> }
```

Comment:
Instead of importing the image, what do you think about pulling it directly alongside the `BlockIcon` component?

`<BlockIcon icon={ 'format-image' } />`

Reply:
Hi @jeherve ! Thanks for your suggestion, applied it :)